app-libs/sgx-runtime/src/lib.rs

```diff
@@ -234,6 +234,10 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = System;
 	type WeightInfo = ();
+	type HoldIdentifier = ();
+	type FreezeIdentifier = ();
+	type MaxHolds = ();
+	type MaxFreezes = ();
```

Comment:
I think this might be it:
> Until you introduce pallets which use holds and freezes, then these can safely be set to ConstU32<0> in the case of the first two and () in the case of the second two.

Source: https://github.com/paritytech/substrate/pull/12951

Let's see if it helps :crossed_fingers: :eyes:.